user_api/serializers.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import fields, serializers, viewsets
+from user_api.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model=User
+        exclude=['id','password','last_login','is_active','staff','admin']
+
+    def create(self,validate_data):
+        return User.objects.create(**validate_data)
+
+    def validate_zipcode(self,value):
+        if len(str(value)) == 6:
```

Comment:
create a constant for 6 and use it over here. Also what if the zipcode is 000000. Is it a valid zipcode ?